### PR TITLE
[Documentation] Prevent unintended LaTeX rendering

### PR DESCRIPTION
**Pull request by mrsegen**
## Summary
Attempt to fix unintended LaTeX rendering between to `$` symbols by applying [MathJaX's ignore class](https://docs.mathjax.org/en/latest/options/document.html?highlight=mathjax_ignore#option-descriptions).

## What's new?
Before this fix, as displayed below, there is awkward rendering. After this fix, the portion of text should now display: `101$, the stop loss will be 91.8$`.
image

### CODE REVIEW

**Comment by mrsegen** [comment]
I’m closing this because I’m pretty sure the fix isn’t as simple as I thought.

**Comment by xmatthias** [comment]
no it's not ...
but it's still a good spot i didn't notice before (that it's rendering wrongly)

**Comment by xmatthias** [comment]
I will reopen this as the fix is actually easier than this (and i've already started doing this on your branch)...

hope you're fine with that :)

**Review by xmatthias — APPROVED** [review]
Once again, great catch! :+1:

**Comment by mrsegen** [comment]
Thanks for picking it up!
